doozer/doozerlib/cli/scan_sources_konflux.py

```diff
@@ -994,6 +1002,144 @@ def is_image_enabled(self, image_name: str) -> bool:
             self.logger.warning('Excluding image %s from the report as it is not enabled in Konflux', image_name)
         return enabled
 
+    async def detect_rhcos_status(self):
+        """
+        gather the existing RHCOS tags and compare them to latest rhcos builds. Also check outdated rpms in builds
+        @return a list of status entries like:
+            {
+                'name': "4.2-x86_64-priv",
+                'changed': False,
+                'reason': "could not find an RHCOS build to sync",
+            }
+        """
+        statuses = []
+
+        version = self.runtime.get_minor_version()
+        primary_container = get_primary_container_name(self.runtime)
+        for arch in self.runtime.arches:
```

Comment:
We might have mixed usage of brew arches (x86_64, aarch64) and go arches (amd64, arm64). I would feel more confident if we make sure with `brew_arch_for_go_arch` or `go_arch_for_brew_arch`.

Reply:
Done with e4feba161127daffab19d838ea7c9fb3ae251e03